### ISSUE TITLE
Support for angular 5

### ISSUE THREAD

**Issue by natcohen** [Stabzs/Angular2-Toaster#136]
Is there a plan to support angular 5? When installing with npm, a warning shows up.

**Comment by dgsqrs** [comment]
+1

**Comment by Stabzs** [comment]
Thanks for opening the issue.  Yes, the library will be rebuilt and re-pinned to Angular 5.  I haven't had time yet but it is a priority for the near future.

**Comment by karclouds** [comment]
Try this. forked and modified version for Angular 5. https://github.com/karclouds/Angular5-Toaster

**Comment by sulhome** [comment]
I think as a workaround if you just import `BrowserAnimationsModule` then it should work for now
I had an issue where the text was not showing on the toast until I hover over it
after adding `BrowserAnimationsModule` to my import the issue seems to disappear.
```typescript
import {BrowserAnimationsModule} from '@angular/platform-browser/animations';
```

**Comment by Stabzs** [comment]
This is published under 4.0.2.  Apologies for the delay.
